feature/policy_forwarding/encapsulation/otg_tests/static_encap_gue_ipv4/README.md

```diff
@@ -0,0 +1,334 @@
+# PF-1.3: Policy-based Static GUE Encapsulation to IPv4 tunnel
+
+## Summary
+
+This test verifies Policy Forwarding (PF) action to encapsulate packets in an
+IPv4 GUE tunnel. The encapsulation is based on a statically configured GUE
+tunnel and static route.
+
+## Testbed Type
+
+*  [`featureprofiles/topologies/ate2_dut5_dum5.testbed`](https://github.com/openconfig/featureprofiles/blob/main/topologies/ate2_dut5_dum5.testbed)
+
+## Procedure
+
+### Test environment setup
+
+*   DUT and DUM have 5 ports each.
+
+    ```
+                              -------                  -------
+                             |       | ==== LAG1 ==== |       |
+          [ ATE:Port1 ] ---- |  DUT  |                |  DUM  | ---- [ ATE:Port2 ]
+                             |       | ==== LAG2 ==== |       |
+                              -------                  -------
+    ```
+
+*   Routes are advertised from ATE:Port2.
+*   Traffic is generated from ATE:Port1.
+*   ATE:Port2 is used as the destination port for GUE encapsulated traffic.
+
+#### Configuration
+
+1.  DUT:Port1 is configured as Singleton IP interface towards ATE:Port1.
+
+2.  DUT:Port2 and DUT:Port3 are configured as LAG1 IP interface towards
+    DUM:Port2 and DUM:Port3 respectively.
+
+3.  DUT:Port4 and DUT:Port5 are configured as LAG2 IP interface towards
+    DUM:Port4 and DUM:Port5 respectively.
+
+4.  DUM:Port1 is configured as Singleton IP interface towards ATE:Port2.
+
+5.  DUT is configured to form two eBGP sessions with DUM using the directly
+    connected LAG interface IPs.
+
+6.  DUM is configured to form an eBGP sessions with ATE:Port2 using the directly
+    connected Singleton interface IP.
+
+7.  ATE:Port2 is configured to advertise destination networks (IPv4-DST-NET,
+    IPv6-DST-NET) and tunnel destination (IPv4-DST-GUE) to DUM. When DUM
+    advertised these prefixes to the DUT over the two eBGP sessions, the
+    protocol next hops for the destination networks should be re-configured as
+    below:
+
+    -   Destination network IPv4-DST-NET with protocol next hop PNH-IPv4.
+    -   Destination network IPv6-DST-NET with protocol next hop PNH-IPv6.
+
+8.  DUT is configured with an IPv4 GUE tunnel with destination IPv4-DST-GUE
+    without any explicit tunnel Type of Service (ToS) or Time to Live (TTL)
+    values.
+
+9.  DUT is configured with the following static routes:
+
+    -   To PNH-IPv4, next hop is the statically configured IPv4 GUE tunnel.
+    -   To PNH-IPv6, next hop is the statically configured IPv4 GUE tunnel.
+
```

Comment:
There are not control protocols running inside this tunnel are there?  (I didn't see any?)

In this case, I recommend we use /network-instances/network-instance/policy-forwarding rules to configure a match and action of encapsulate.  (New paths needed for the GUE encapsulate action as you noted below).

Reply:
Correct, there are no control protocols running inside this tunnel. I'll follow the OC example you mentioned for the new GUE paths.